awx/api/serializers.py

```diff
@@ -667,6 +668,11 @@ def get_types(self):
         else:
             return super(UnifiedJobSerializer, self).get_types()
 
+    def get_events_processed(self, obj):
```

Comment:
Incorporate the timeout that's in the view here ? Return `True` if timeout reached.